src/main/java/com/plivo/api/models/campaign/Campaign.java

```diff
@@ -118,6 +120,10 @@ public String getHelpKeywords(){
     return this.helpKeywords;
   }
 
+  public String getCampaignSource(){
```

Comment:
Has to be added in camapignResponse.java as well. 
campaign_source is not appearing currently in the GET campaign response